platform/linux-generic/m4/odp_crypto.m4

```diff
@@ -0,0 +1,36 @@
+# ODP_CRYPTO
+# ----------
+# Select default crypto implementation
+AC_ARG_ENABLE([crypto-default],
+             [AS_HELP_STRING([--enable-crypto-default],
+                             [choose default crypto implementation [openssl/armv8crypto/null] (linux-generic)])],
+             [default_crypto=$enableval], [default_crypto=openssl])
+
+AS_IF([test "x$default_crypto" != "xopenssl" -a "x$default_crypto" != "xarmv8crypto" -a "x$default_crypto" != "xnull"],
+      [AC_MSG_ERROR([Invalid crypto implementation name])])
+
+AS_IF([test "${with_openssl}" = "no"],
+      [default_crypto=null])
+
+##########################################################################
+# OpenSSL implementation
+##########################################################################
+AS_IF([test "x$default_crypto" == "xopenssl"],
+      [ODP_OPENSSL
+       have_openssl=1], [have_openssl=0])
```

Comment:
This macro is already called in `platform/linux-generic/m4/odp_openssl.m4`, so this statement could be removed. There is a warning: `Strong cryptography is not available without OpenSSL` in `odp_openssl.m4`, which could also be removed now.

Reply:
I will make the change. Thanks Matias!